modules/i3s/src/i3s-slpk-loader.ts

```diff
@@ -0,0 +1,22 @@
+import {LoaderOptions, LoaderWithParser} from '@loaders.gl/loader-utils';
+import {parseSlpk} from './lib/parsers/parse-slpk';
```

Comment:
- I would prefer keeping this capitalized `parseSLPK`, `SLPKLoader`, `SLPKLoaderOptions` ...